src/views/Run/internal/ResultDisplay/internal/ChartsSection/internal/BarChart/internal/BarChart.helpers.tsx

```diff
@@ -0,0 +1,18 @@
+import s from './bar-chart.module.scss'
+import { IProps } from './BarChart.types'
+
+export const getClassNames = ({ className }: Pick<IProps, 'className'>) => {
+  const classNames: string[] = [
+    s['barChart'],
+    // 'f',
+    // 'f-direction-row',
+    // 'f-ai-center',
+    // 'p-3',
```

Comment:
We should delete this comments if they are not usefull

Reply:
Oh thanks, I forgot this!